src/NotifierCLI.hpp

```diff
@@ -50,18 +49,13 @@ class CLI
     void jump(const std::string& options);
     void next(const std::string& options);
     void previous(const std::string& options);
-    void remove(const std::string& options);
     void help(const std::string& options);
     void list(const std::string& options);
     void quit(const std::string& options);
 
 private:
-
-    // TODO dont execute anything if db is empty
-    // TODO maybe store as a reference
-    std::map<unsigned, Notification> _db;
-
-    std::map<unsigned, Notification>::iterator _iter;
+    const std::map<unsigned, Notification>& _db;
```

Comment:
Whole database is stored only by reference... so the map has to exist in mainCLI function for this to work which currently is not a problem... this eliminates one copy of whole notification db